src/sentry/dynamic_sampling/lastest_release_booster.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+from django.core.cache import cache
```

Comment:
We should use a redis client rather than using `cache`. `django.core.cache.cache` is a memcache in prod and can lose data. Redis will be persistent. We also need a redis client for some of the changes further down I would like to see.